### ISSUE TITLE
Field validation design is looking bad

### ISSUE THREAD

**Issue by DrXSsive** [SamProf/MatBlazor#620]
Hi,

I just updated my project from 2.3.0 to 2.6.0 and now the field validations look awful. Here is snapshot:

![Issue](https://user-images.githubusercontent.com/40558386/85279369-607baa00-b497-11ea-8af0-fd840bb6fcc3.PNG)

Any work around to move it back to the previous design?

Regards
Jawwad

**Comment by enkodellc** [comment]
Yes that doesn't look great. I suggest to do a diff on the versions and find out what changed and offer up a PR to fix it.
